docs/content/configuration/prefill-disk-cache.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Pre-filling the Disk Cache
+description: How to pre-fill the disk cache
+weight: 4
+---
+
+One of the popular features of Athens is that it can be run completely cut off to the internet. In this case, though, it can't reach out to an upstream (e.g. a VCS or another module proxy) to fetch modules that it doesn't have in storage. So, we need to manually fill up the disk partition that Athens uses with the dependencies that we need.
+
+This document will guide you through packaging up a single module called `github.com/my/module`, and inserting it into the Athens disk storage.
+
+# First, get the tools
+
+You'll need to produce the following assets from module source code:
+
+- `source.zip` - just the Go source code, packaged in a zip file
```

Comment:
I mentioned it below as well, but the downloaded `zip` when fetching a module is $VERSION as well (e.g. `v1.0.0.zip`)

Reply:
@jpreese what I meant to say here is that Athens will need a `source.zip` in its disk storage location. Not sure if that was clear?